src/index.html

```diff
@@ -1,17 +1,210 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" name="personal-information">
+
+      <fieldset class="form-field personal-information">
+        <legend>Personal information</legend>
+        <div class="input surname">
+          <label for="surname">Surname:
```

Comment:
Move this Surname to the next line and fix input formatting according to the checklist:
```html
<label for="surname">
  Surname:

  <input
    type="text"
    autocomplete="off"
    id="surname"
    name="surname"
  >
```